release.py

```diff
@@ -256,11 +260,105 @@ def command_stage_docs():
 
     sys.exit(0)
 
+def command_release_announcement_email():
+    release_version_num = raw_input('What is the current release version:')
```

Comment:
Should we do validation here like we do in the default staging command, i.e. like this https://github.com/apache/kafka/blob/trunk/release.py#L330-L338? Maybe we can refactor the validation/setting of global variable `release_version_parts` there so we can just reuse that bit of code?

Reply:
yes, refactored the code